cmd/server/main.go

```diff
@@ -111,11 +256,58 @@ func handleGetValue(s *MemStorage) http.HandlerFunc {
 	}
 }
 
+func handleGetValueJSON(s *MemStorage) http.HandlerFunc {
```

Comment:
[CHANGE REQUEST] Please split your application by Tiers.
https://medium.com/@shrestha.matina.20/what-is-the-3-tier-architecture-4520522e0720